group1/comm/static/comm_tool.css

```diff
@@ -32,16 +32,21 @@ h4 {
 
 header {
   width:100%;
-  height:7rem;
+  /*height:7rem;*/
```

Comment:
Is there a reason to keep these commented out lines in the code?

Reply:
No reason. They can be removed.